src/calendar-app/calendar/gui/EditCalendarDialog.ts

```diff
@@ -48,26 +63,52 @@ export function showEditCalendarDialog(
 							colorStream(target.value)
 						},
 					}),
-					!shared &&
-						m(RemindersEditor, {
-							alarms,
-							addAlarm: (alarm: AlarmInterval) => {
-								alarms?.push(alarm)
-							},
-							removeAlarm: (alarm: AlarmInterval) => {
-								const index = alarms?.findIndex((a: AlarmInterval) => deepEqual(a, alarm))
-								if (index !== -1) alarms?.splice(index, 1)
-							},
-							label: "calendarDefaultReminder_label",
-						}),
+					!shared && calendarType !== CalendarType.URL
```

Comment:
could this be wrapped in a more descriptive function like `isPrivateCalendar() `? (That is what this check is, right?)
Or alternatively, would it be possible to add this to the `CalendarType` so you'd have types `URL`, `SHARED`, and `PRIVATE`?

Reply:
It checks if the calendar is private but also if it is not an external calendar we subscribed too.
External calendars are also private and owned by the user.
I will extract the externalCalendar comparison to a function.